backend/src/main/java/wooteco/retrospective/presentation/member/MemberController.java

```diff
@@ -26,6 +26,13 @@ public MemberController(MemberService memberService) {
     public ResponseEntity<MemberLoginResponse> loginMember(@Valid @RequestBody MemberLoginRequest request) {
         MemberLoginDto requestDto = new MemberLoginDto(request.getName());
         MemberLoginResponse response = new MemberLoginResponse(memberService.loginMember(requestDto).getToken());
```

Comment:
```suggestion
        String token = memberService.loginMember(requestDto).getToken()
        MemberLoginResponse response = new MemberLoginResponse(token);
```
이런 로직은 분리가 돼 있어야지 가독성이 좋다고 생각합니다.